op-node/rollup/derive/attributes.go

```diff
@@ -84,7 +88,9 @@ func (ba *FetchingAttributesBuilder) PreparePayloadAttributes(ctx context.Contex
 		if l2Parent.L1Origin.Hash != epoch.Hash {
 			return nil, NewResetError(fmt.Errorf("cannot create new block with L1 origin %s in conflict with L1 origin %s", epoch, l2Parent.L1Origin))
 		}
+		start := time.Now()
 		info, err := ba.l1.InfoByHash(ctx, epoch.Hash)
+		log.Debug("FetchingAttributesBuilder:PreparePayloadAttributes: called ba.l1.InfoByHash.", "duration", time.Since(start))
```

Comment:
```suggestion
		log.Debug("derivation fetcher performance info", "stage", "FetchingAttribtuesBuilder", "caller", "PreparePayloadAttributes", "method", "InfoByHash", "duration", time.Since(start))
```